package.json

```diff
@@ -2,7 +2,7 @@
   "name": "mongoman",
   "author": "John Hofrichter <https://github.com/johnhof>",
   "description": "MongoDB management utility",
-  "version": "0.0.4",
+  "version": "1.0.0",
```

Comment:
ah yea, my bad. It should technically be 1.1.0 since its a new set of features... not sure how the version is still 0.0.4 thats a definite oversight...